lib/lightning_web/router.ex

```diff
@@ -176,7 +176,8 @@ defmodule LightningWeb.Router do
       live "/profile/tokens", TokensLive.Index, :index
       live "/profile/tokens/:id/delete", TokensLive.Index, :delete
 
-      live "/", DashboardLive.Index, :index
+      get "/", Plugs.Redirect, to: "/projects"
+      live "/projects", DashboardLive.Index, :index
```

Comment:
This needs to be configurable to use a different Liveview and modal on Thunderbolt.

Reply:
This is now handled. I moved the routing to `/projects` to the `config.exs` so that Thunderbolt would be able to override it.